### PR TITLE
Fix editable condition/marker for capabilities add/update

### PR DESCRIPTION
**Pull request by adreghiciu**

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/capability/CapabilitySettingsForm.js [hunk 1]
```diff
@@ -50,6 +50,18 @@ Ext.define('NX.coreui.view.capability.CapabilitySettingsForm', {
 
   /**
    * @override
+   */
+  initComponent: function() {
+    var me = this;
+
+    me.editableCondition = me.editableCondition || NX.Conditions.isPermitted('nexus:capabilities:update');
```
Comment (djsauble) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/capability/CapabilitySettingsForm.js]: Is the default condition ever overridden? If not, why the `||`?

Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/capability/CapabilitySettingsForm.js]: It is overridden in CapabilityAdd

Comment (djsauble) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/capability/CapabilitySettingsForm.js]: k, +1 from me

**Comment by kellyrob99** [comment]
+1, provided #1411 doesn't conflict

**Pull request merged**